site/docs/guides/evaling-with-harmbench.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_label: Evaluating LLM safety with HarmBench
+---
+
+# Evaluating LLM safety with HarmBench
+
+Recent research has shown that even the most advanced LLMs [remain vulnerable](https://unit42.paloaltonetworks.com/jailbreaking-deepseek-three-techniques/) to adversarial attacks. Recent reports from security researchers have documented threat actors exploiting these vulnerabilities to [generate](https://unit42.paloaltonetworks.com/using-llms-obfuscate-malicious-javascript/) [malware](https://www.proofpoint.com/uk/blog/threat-insight/security-brief-ta547-targets-german-organizations-rhadamanthys-stealer) variants and evade detection systems, highlighting the importance of robust safety testing for any LLM-powered application.
+
+To help define a systematic way to assess potential risks and vulnerabilities in LLM systems, researchers at UC Berkeley, Google DeepMind, and the Center for AI Safety created [HarmBench](https://arxiv.org/abs/2402.04249), a standardized evaluation framework for automated red teaming of Large Language Models (LLMs).
```

Comment:
might be worth mentioning the harm areas it covers

Reply:
We did include in the plugin doc, but I think I will also just duplicate here.